### ISSUE TITLE
Console warning: forcedUpdate

### ISSUE THREAD

**Issue by t1mmen** [eiriklv/react-masonry-component#1]
Hi,

Thanks for this component! I'm having a slight issue when navigating away from pages using this component, with the console giving me the following warning:

```
Warning: forceUpdate(...): Can only update a mounted or mounting component. This usually means you called forceUpdate() on an unmounted component. This is a no-op.
```

**Comment by t1mmen** [comment]
I've tried removing the `forcedUpdate` call in the `componentWillRecieveProps` hook, which fixed the warnings issue. Everything still appears to be working as expected, but I'm thinking you probably had a good reason to put this call in?

**Comment by eiriklv** [comment]
Hi @t1mmen! There's a reason for it (there's some quirks when you use infinite scrolling/reloading of entire collections in certain cases). But to mitigate the warning I've released a fix that checks if the component is mounted (`this.isMounted`) before doing `this.forceUpdate`. Released as 1.0.3 :-)
